openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
Can you please refactoring the script that it has more have a look and feel of a C programm with a main and help function?
Have a look at https://github.com/openwrt/packages/blob/master/net/mwan3/files/usr/sbin/mwan3rtmon thats only an example.
From my point of view, this has certain advantages. If you later decide to make a C program out of it, then you already know what you need for functions. My experience as a maintainer has often shown me that.

Also add a valid license header

Reply:
@feckert I think improvements like this one be done incrementally, so this request can become an issue and we can work on it once we the packages compile and the basic functionality works as expected.

Reply:
Opened #33 for this